templates/3b_Generic_Cluster_Cells.ipynb

```diff
@@ -0,0 +1,665 @@
+{
```

Comment:
<p>Anything that says pixel SOM/meta cluster in this notebook is not relevant anymore, and should be updated to state what's actually being computed if its relevant, or removed if its not. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/892/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>